tests/Set/SyliusPlus/B2BSuite/config/configured_rule.php

```diff
@@ -7,5 +7,8 @@
 
 return static function (RectorConfig $rectorConfig): void {
     $rectorConfig->import(__DIR__ . '/../../../../../config/config.php');
-    $rectorConfig->sets([SyliusPlus::B2B_SUITE]);
+    $rectorConfig->sets([
+        SyliusPlus::B2B_SUITE_20,
```

Comment:
We also used this suite to version 1.0 of B2B, so I would be for leaving `B2B_SUITE` name 🤔 
```suggestion
        SyliusPlus::B2B_SUITE,
```